lib/distributions.sh

```diff
@@ -108,6 +108,17 @@ install_common()
 	# force change root password at first login
 	chroot "${SDCARD}" /bin/bash -c "chage -d 0 root"
 
+	# change console welcome text
+	echo -e "${DISTRIBUTION} ${RELEASE^} with Armbian Linux \\l \n" > $SDCARD/etc/issue
+	echo "${DISTRIBUTION} ${RELEASE^} with Armbian Linux" > $SDCARD/etc/issue.net
+
+	# enable few bash aliases enabled in Ubuntu by default to make it even
+	sed "s/#alias ll='ls -l'/alias ll='ls -l'/" -i $SDCARD/etc/skel/.bashrc
```

Comment:
consider using sed -e (this way you can also put multiple sed calls into a single command execution)